xray/client/xray_viewer.ts

```diff
@@ -63,15 +71,16 @@ export class XRayViewer {
   private nodes: {[key: string]: NodeData} = {};
   private currentlyLoading: number;
   private nodesToLoad: string[] = [];
+  private currentForLevelQueryNumber: number;
+  private displayLevel: number;
   private meta: any; // Will be undefined until we have loaded the metadata.
+
   constructor(private scene: THREE.Scene, private prefix: string) {
     this.currentlyLoading = 0;
-    const request = new Request(`${this.prefix}/meta`, {
-      method: 'GET',
-      credentials: 'same-origin',
-    });
+    this.currentForLevelQueryNumber = 0;
```

Comment:
I find this name confusing. I don't what the adjective "current" applies to — the level? The number?

Reply:
I renamed it to `nextNodesForLevelQueryId`.